src/top/state.rs

```diff
@@ -50,10 +52,11 @@ pub async fn updater(mut state: State, mut event_rx: EventRx) -> StateRx {
                             }
                         }
                     }
-                    EventType::ProcessedEventsThroughputs(rows) => {
+                    EventType::ProcessedEventsThroughputs(interval, rows) => {
```

Comment:
Nit: I might call these `interval_ms` just for clarity (or change `interval` to a `Duration`).

Reply:
This models the `interval` in the API schema (an `Int!`), and the `--interval` CLI switch. I'd normally pass around as a duration, but in this case, it'd be an i64 -> Duration -> i64, for the sake of a cleaner signature. I think a comment is probably sufficient.